StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Link/PayWithLinkController.swift

```diff
@@ -36,7 +38,11 @@ final class PayWithLinkController {
             let keyWindow = UIApplication.shared.stp_hackilyFumbleAroundUntilYouFindAKeyWindow(),
             let presentedViewController = keyWindow.findTopMostPresentedViewController()
         else {
-            assertionFailure("No key window with view controller found")
+            let errorAnalytic = ErrorAnalytic(event: .unexpectedPaymentSheetError,
+                                              error: Error.noWindowWithViewController)
+            STPAnalyticsClient.sharedClient.log(analytic: errorAnalytic)
+            stpAssertionFailure("No key window with view controller found")
+            completion(.failed(error: PaymentSheetError.unableToPresent), nil)
             return
```

Comment:
Why do we use this method?  PaymentSheet always has a view controller to present on.  Is it difficult to refactor PaymentSheet to use `present(from:completion:)` below, instead, and delete this method + the error code? It looks like we can just pass authenticationContext.presentingViewController